mat-fhir-services/src/main/java/gov/cms/mat/fhir/services/components/fhir/FhirIncludeLibraryProcessor.java

```diff
@@ -51,40 +56,55 @@ private void checkIsFhir(CqlTextParser cqlTextParser) {
         }
     }
 
-
     private FhirIncludeLibraryResult processParser(CqlTextParser cqlTextParser) {
-        FhirIncludeLibraryResult res = new FhirIncludeLibraryResult();
+        var fhirIncludeLibraryResult = new FhirIncludeLibraryResult();
 
         boolean result = true;
 
         LibraryProperties libraryProperties = cqlTextParser.getLibrary();
 
-        res.setLibraryName(libraryProperties.getName());
-        res.setLibraryVersion(libraryProperties.getVersion());
+        fhirIncludeLibraryResult.setLibraryName(libraryProperties.getName());
+        fhirIncludeLibraryResult.setLibraryVersion(libraryProperties.getVersion());
+
+        getIncludedLibraryReferences(cqlTextParser, fhirIncludeLibraryResult, result);
```

Comment:
The `boolean result` will not be modified. Instead, how about updating `outcome` directly on the `FhirIncludeLibraryResult` object?
```suggestion
        getIncludedLibraryReferences(cqlTextParser, fhirIncludeLibraryResult);
```